packages/dapp-svelte-wallet/api/src/wallet.js

```diff
@@ -69,7 +73,9 @@ export function buildRootObject(_vatPowers) {
     },
   });
 
-  async function startup({ zoe, board }) {
+  async function startup({ zoe, board, rendezvous: chainRendezvous }) {
+    rendezvous = chainRendezvous;
```

Comment:
huh, why rename and then rename again?

Reply:
Clarified.  I needed to assign it to a `let rendezvous` variable that would otherwise have the same name.